### PR TITLE
Add Styling

### PR DESCRIPTION
**Pull request by Redforce04**

### CODE REVIEW

File: LCAPI.Core/LCAPI.Core.csproj [hunk 1]
```diff
@@ -1,22 +1,53 @@
...
+        <PackageReference Include="UnityEngine.Modules" Version="2022.3.9" IncludeAssets="compile" ExcludeAssets="RUNTIME" />
     </ItemGroup>
 
     <ItemGroup Condition="'$(TargetFramework.TrimEnd(`0123456789`))' == 'net'">
-        <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="all"/>
+        <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="all" ExcludeAssets="RUNTIME" />
     </ItemGroup>
+    
+    <ItemGroup>
+        <Reference Include="Assembly-CSharp" HintPath="$(LC_REFERENCES)\Assembly-CSharp-Publicized.dll" Private="False" />
```
Comment (MaxWasUnavailable) [inline LCAPI.Core/LCAPI.Core.csproj:29]: Let's keep this to the default Assembly-CSharp for now. Publicised dll can be sorted out once the rest of the team is onboard with it.

File: LCAPI.Core/LCAPI.Core.csproj [hunk 2]
```diff
@@ -1,22 +1,53 @@
...
+        <Reference Include="Assembly-CSharp-firstpass" HintPath="$(LC_REFERENCES)\Assembly-CSharp-firstpass.dll" Private="False" />
```
Comment (MaxWasUnavailable) [inline LCAPI.Core/LCAPI.Core.csproj:30]: Let's keep unnecessary references out of the csproj for now. Commits that add functionalty can add them.

Comment (MaxWasUnavailable) [reply LCAPI.Core/LCAPI.Core.csproj:30]: Applies to references following this one, as well.

**Review by MaxWasUnavailable — CHANGES_REQUESTED** [review]
Found a couple of small required changes. Looks fine beyond that.

**Review by Redforce04 — COMMENTED** [review]
Fixed with latest PR. It actually builds to the right directory now, as well.

**Review by MaxWasUnavailable — APPROVED** [review]
Looks good! Ready to merge

**Pull request merged**